guide/C/fdl-appendix.xml

```diff
@@ -1,566 +1,467 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE appendix SYSTEM "gnc-docbookx.dtd">
-<!--
-     The GNU Free Documentation License 1.1 in DocBook
```

Comment:
Current fdl-appendix.xml changes the section number.
When we render it using docbook-xsl, `0. PREAMBLE` is changed to `E.0  PREAMBLE`.

FDL says
```
  <simpara>
    Everyone is permitted to copy and distribute verbatim copies of this
    license document, but changing it is not allowed.
  </simpara>
```

I don't like to change the section number.
In fact, the Docbook version of FDL 1.3 uses this method of tagging.


While I saw 
https://www.gnu.org/licenses/fdl-1.3.dbk,
I added directly DocBook tags into 
https://www.gnu.org/licenses/old-licenses/fdl-1.1.txt.

I didn't notice the comment part of the original file.

Reply:
> I didn't notice the comment part of the original file.
> GNOME developers' works are not introduced in the new fdl-appendix.xml.

I suspect, gnu.org never offered a docbook4 version of fdl1.1 and our ancestors made extensive use of templates from the GNOME Documentation Project  And GDP converted fdl from txt to dbk and collected 'inofficial transslations'.

For later versions, GNU offers docbook variants while GNOME still stays at 1.1 because for new docs they use a CC variant. But both projects are using now `<!DOCTYPE article` (instead of appendix).

Because we are short before a release, i would recommend to stay with GNOME's fdl 1.1 flavour.

Together with your other renaming we should have then
manual
guide
fdl

But you are free to start a license discussion on gnucash-devel.

BTW: If you move files, are you using `git mv <source> <destination>` to keep the history?

Reply:
> BTW: If you move files, are you using git mv <source> <destination> to keep the history?

Yes I am.